libs/sdk-ui-tests/stories/visual-regression/kit/WhiteLabeling/WhiteLabeling.tsx

```diff
@@ -0,0 +1,41 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { storiesOf } from "../../../_infra/storyRepository";
+import { UiKit } from "../../../_infra/storyGroups";
+import { wrapWithTheme } from "../../themeWrapper";
+import { WhiteLabeling } from "@gooddata/sdk-ui-kit";
+
+const WhiteLabelingTest = () => {
```

Comment:
Please rename this story according to the newly generic component. The texts that talks about white-labeling are OK as is 👍